examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/WildcardFragment.kt

```diff
@@ -179,7 +179,8 @@ class WildcardFragment : Fragment() {
                                        minInterval,
                                        maxInterval,
                                        keepSubscriptions,
-                                       isFabricFiltered)
+                                       isFabricFiltered,
+                                       0)
```

Comment:
Can you add inline comments for these zeroes? e.g.

`/* imTimeoutMs= */ 0`

here and below? That will make it clearer what that zero represents.

Reply:
done, thanks